protected/config/core/common.php

```diff
@@ -204,7 +204,7 @@
             'class' => 'ModuleAPI',
         ),
         'request' => array(
-            'enableCsrfValidation' => true,
+            'enableCsrfValidation' => false,
```

Comment:
We may need to revisit this if we determine that CSRF validation is necessary for security purposes.